### ISSUE TITLE
v2.0.1 does not have a `.whl` file and `tar` file may be incorrect.

### ISSUE THREAD

**Issue by hrshdhgd** [richmilne/JaroWinkler#5]
Hello,

We have a project whose dependency is your project. On running an install of packages, I experienced this error:

```
• Installing jaro-winkler (2.0.1): Failed

ERROR: file:///Users/.../Caches/.../jaro_winkler-2.0.1.linux-x86_64.tar.gz does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.
```

I looked at the [pypi release here](https://pypi.org/project/jaro-winkler/2.0.1/) and noted:
1.  The `.whl` is missing.
2. The `tar.gz` file on being unzipped reveals directories that are not similar to those in v2.0.0. 

My guess is something may have gotten messed up during the release process. For now we have anchored the project requirements to use v2.0.0 and all is well, but I just thought of letting you know about this issue.

If I'm mistaken, please let me know. Thanks!

**Comment by richmilne** [comment]
You're very likely correct! ;) Thanks for letting me know.

I've re-created the package, as a python wheel, and I've uploaded it to the test pypi repository. I've also bumped the version number to 2.0.2, to distinguish this package from the others.

Can you try install this package (`pip install -i https://test.pypi.org/simple/ jaro-winkler`) and let me know if this release resolves the problem? If so, let me know, and I'll push it to main pypi repository.

Cheers,
Richard

**Comment by hrshdhgd** [comment]
I tried `pip install url-to-the-v2.0.2-whl-file` and it worked fine. I looked into the `.whl` as well and that seems to be as expected. I think v2.0.2 is good to go!
Thank you for looking into this!

**Comment by richmilne** [comment]
Now fixed and published as v2.0.3 (as I'd also made a small fix to the c source-file in the interim).

**Comment by hrshdhgd** [comment]
Thank you very much Richard!
